spark-3.5.yaml

```diff
@@ -1,18 +1,20 @@
 package:
   name: spark-3.5
   version: 3.5.1
-  epoch: 7
+  epoch: 8
   description: Unified engine for large-scale data analytics
   copyright:
     - license: Apache-2.0
   dependencies:
     provides:
       - spark=${{package.full-version}}
     runtime:
+      - R
+      # - python3: Python dependency will be added in the image, since upstream supports different Python versions starting from 3.9.
       - bash
       - busybox
-      - openjdk-17
-      - openjdk-17-default-jvm
+      # - openjdk-17: OpenJDK is required for running Spark, but will be added in the image, since upstream supports different Java versions.
```

Comment:
nit: we should remove these versus commenting them out

Reply:
I was going to remove it but then thought that someone may bring those back in the future, so wanted to drop as a warning actually. Should we remove those anyway?

Reply:
We still wouldn't want to add back the full JDK though. I'd :axe: it

Reply:
Done. Removed all commented out packages but feel a bit uncomfortable actually, i'd be nice if we should drop a comment regarding to why we shouldn't have `python` and `openjdk-` package in the runtime deps? WDYT?

Reply:
That could certainly be helpful for anyone looking at this in the future. I'd suspect we'll have more java packages where this is the case